src/backend/src/controllers/projects.controllers.ts

```diff
@@ -26,12 +26,53 @@ export default class ProjectsController {
     }
   }
 
+  // static async createProject(req: Request, res: Response, next: NextFunction) {
+  //   try {
+  //     const user: User = await getCurrentUser(res);
+  //     const { crId, carNumber, name, summary, teamId } = req.body;
+
+  //     const createdWbsNumber: WbsNumber = await ProjectsService.createProject(user, crId, carNumber, name, summary, teamId);
+
+  //     return res.status(200).json(wbsPipe(createdWbsNumber));
+  //   } catch (error: unknown) {
+  //     next(error);
+  //   }
+  // }
   static async createProject(req: Request, res: Response, next: NextFunction) {
     try {
       const user: User = await getCurrentUser(res);
-      const { crId, carNumber, name, summary, teamId } = req.body;
+      const {
+        name,
+        crId,
+        carNumber,
+        teamIds,
+        budget,
+        summary,
+        projectLeadId,
+        projectManagerId,
+        links,
+        goals,
+        features,
+        otherConstraints,
+        rules
+      } = req.body;
 
-      const createdWbsNumber: WbsNumber = await ProjectsService.createProject(user, crId, carNumber, name, summary, teamId);
+      const createdWbsNumber: WbsNumber = await ProjectsService.createProject(
+        user,
+        crId,
+        carNumber,
+        name,
+        summary,
+        teamIds,
+        budget || null,
+        links || null,
+        rules || null,
```

Comment:
shouldn't these just be empty arrays now when not filled in by the user now, so not '|| null'